maptk/local_geo_cs.cxx

```diff
@@ -123,22 +128,17 @@ local_geo_cs
 ::update_metadata(vital::simple_camera const& cam,
                   vital::video_metadata& md) const
 {
-  if( !geo_map_algo_ )
-  {
-    return;
-  }
   double yaw, pitch, roll;
   cam.rotation().get_yaw_pitch_roll(yaw, pitch, roll);
   yaw *= rad2deg;
   pitch *= rad2deg;
   roll *= rad2deg;
-  vital::vector_3d c = cam.get_center() + utm_origin_;
-  double lat, lon;
-  geo_map_algo_->utm_to_latlon(c.x(), c.y(), utm_origin_zone_, true,
-                               lat, lon);
+  vital::vector_3d c = cam.get_center();
+  vital::geo_point gc(vector_2d(c.x(), c.y()) + geo_origin_.location(),
+                      geo_origin_.crs());
+  vector_2d lon_lat = gc.location(SRID::lat_lon_WGS84);
 
-  vital::geo_lat_lon latlon( lat, lon );
-  md.add( NEW_METADATA_ITEM( VITAL_META_SENSOR_LOCATION, latlon ) );
+  md.add( NEW_METADATA_ITEM( VITAL_META_SENSOR_LOCATION, lon_lat ) );
```

Comment:
FYI, it may or may not be desirable, but plausibly you could set this to just the geo_point, in whatever CRS. If we're doing it right, the consumer is responsible for requesting CRS conversion...